jobs/iptables-manager/templates/bin/iptables-manager_ctl.erb

```diff
@@ -22,8 +22,12 @@ exec 2>> $LOG_DIR/$JOB_NAME.stderr.log
 case $1 in
   start)
     iptables -F #Flush all existing rules
+    iptables -A INPUT -p ICMP --icmp-type 8 -s  "<%= p('allow_ips_list') %>" -j ACCEPT
+    iptables -A INPUT -p udp -s "<%= p('allow_ips_list') %>" -j ACCEPT
     iptables -A INPUT -p tcp -s  "<%= p('allow_ips_list') %>" -j ACCEPT
     <% if_p('block_ips_list') do |block_ips_list| %>
+    iptables -A INPUT -p ICMP --icmp-type 8 -s "<%= block_ips_list %>" -j  DROP
```

Comment:
Can you please elaborate on `block_ips_list` here ? Anything outside acceptance CIDR ranges is not meant to be blocked ?

Reply:
Yes. Since this is to ensure we isolate within the SF services subnet, SF tells what IPs to allow and what ips/CIDR to block from the same.

Reply:
Agreed. But will this mean that these rules will have to be updated as and when any new range is introduced in the landscape ? A blanket deny if nothing is accepted will not work here ? 

Please let me know if I am missing something

Reply:
No we that will never be the case as we will always enable the traffic for these protocols in the cluster, so allow list will never be empty. 
By new range you mean new subnets right? Across subnet if we are to restrict traffic we anyways have security group rules which is the right place to address such restrictions. Iptables rules are more from within the the same network for isolation across clusters.